examples/sdcard/console/grandcentral-m4.go

```diff
@@ -1,4 +1,5 @@
-// +build grandcentral-m4
+//go:build ignore
+// +build ignore
```

Comment:
It looks like this is a bug: the build tag should be `grandcentral_m4` (dashes are not allowed).

Reply:
Corrected.